l10n_pa_withholding/model/account_invoice.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+from openerp import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    def wh_tax_account(self, line):
+        """Retrieves the account to use as Withholding Account"""
+        return line.account_id.id
+
+    @api.model
+    def wh_move_line_get_item(self, line):
+        sign = 1
+        if line.invoice_id.type in ('out_invoice', 'in_invoice'):
+            sign = -1
+        return {
+            'type': 'src',
+            'name': line.name.split('\n')[0][:64],
+            'account_id': self.wh_tax_account(line),
+            'price': sign * line.amount,  # TODO: Insert Withheld value here!
+            'tax_code_id': line.tax_code_id.id,
+            'tax_amount': line.amount,
+        }
+
+    @api.model
+    def wh_move_line_get(self):
+        """Creates Withholding for taxes in invoice"""
```

Comment:
THe agreement was create an extra account move, not an extra tax.line, why are you coding an extra tax.line?